### PR TITLE
fix: #26 add global language and wifi disabled icons

### LINKED ISSUE #26: Global language and Wifi disabled icons
# Add new icon(s) to the library

> Please list the icon(s) that need to be added to the library

1. Global language globe icon
2. Wifi unavailable/missing icon

## Icon art

> The icon(s) per this request, has the new art been completed?

- [x] Yes
- [ ] No

> If YES, does the new icon art follow all the recommendations from the [Auro Icon Design Guidlines](https://auro.alaskaair.com/foundations/iconography/design-guidelines)?

- [x] Yes
- [ ] No

> If YES, the icon(s) in the request, have they been added to the UiKit Abstract repo?

- [x] Yes
- [ ] No

### Attach exported SVG files to this issue

[globe and wifi disabled.zip](https://github.com/AlaskaAirlines/OrionIcons/files/4377390/globe.and.wifi.disabled.zip)

[globe and wifi disabled.zip](https://github.com/AlaskaAirlines/OrionIcons/files/4377394/globe.and.wifi.disabled.zip)

See the options below for attaching a file to this request.

### PR DESCRIPTION
**Pull request by dlesaca**
# Alaska Airlines Pull Request

Included in this PR is the addition of two icons: `global-language.svg` and `wifi-disabled.svg`. This PR will address and will close #26 once merged.

**Fixes:** # (issue, if applicable)

Fixes #26 

## Summary:

Before these changes, these icons did no exist in the icon system.

## Type of change:

Please delete options that are not relevant.

- [x] New capability 

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._ 

**Thank you for your submission!**<br>
-- Orion Design System Team

### CODE REVIEW

File: src/data/icons.json [hunk 1]
```diff
@@ -338,6 +338,13 @@
       "style": "ico_squareLarge",
       "category": "interface"
     },
+    {
+      "title": "Global Language",
```
Comment (blackfalcon) [inline src/data/icons.json]: Why did you fill out all the things up here, but not in lines 564 and 565?

This also opened my eyes to a complex issue with the primaryBuild.js file. 
[Functionality is confusing #28](https://github.com/AlaskaAirlines/OrionIcons/issues/28)

Comment (dlesaca) [reply src/data/icons.json]: Honestly, I was following the previous pattern set forth by other items in the category. Would you like me to fill it out?

Comment (blackfalcon) [reply src/data/icons.json]: That is what I was afraid of. There are real issues that were missed when all the functionality of this thing blew up. 

Yes, please fill out the rest and other clean up will happen with #28

**Pull request merged**